### ISSUE TITLE
Silent Hill 2 Remake plugins need to be updated for the new Patch 1.04

### ISSUE THREAD

**Issue by UnknownUserMeMe** [RoseTheFlower/UltrawideIndex#88]
https://steamcommunity.com/games/2124490/announcements/detail/4538031421716523317

**Comment by roguen1** [comment]
Idk if someone is having this issue, but I'm experiencing heavy stuttering in the first cutscene of the game (james in bathroom) when using this mod with the latest patch of the game, removing the mod fixed it and runs smooth again.

**Comment by RoseTheFlower** [comment]
Works fine on my end. #83

**Comment by roguen1** [comment]
> Works fine on my end. #83

yeah, srry, it wasn't your mod, it was the bad implementation of FSR3 FG that they did with today's patch, if I use FG along with your mod, it stutters like hell in some scenarios, but this mod still works well, they need to fix this bad FG
